src/Shared/UnitTests/FileUtilities_Tests.cs

```diff
@@ -329,38 +330,20 @@ public void CheckDerivableItemSpecModifiers()
             Assert.False(FileUtilities.ItemSpecModifiers.IsDerivableItemSpecModifier("recursivedir"));
         }
 
-        [Fact(Skip = "Test fails in xunit when multiple tests are run")]
+        [Fact]
         public void GetExecutablePath()
         {
-            string path;
-
-            // If FileUtilities knows we are running tests, it will return the assembly path, not the
-            // module path
-            if (FileUtilities.RunningTests)
-            {
-                path =
-                    Path.Combine(
-                        Path.GetDirectoryName(FileUtilities.ExecutingAssemblyPath)
-                            .TrimEnd(new[] { Path.DirectorySeparatorChar, Path.AltDirectorySeparatorChar }),
-                        "MSBuild.exe");
-            }
-            else
-            {
-                StringBuilder sb = new StringBuilder(NativeMethodsShared.MAX_PATH);
-#if FEATURE_HANDLEREF
-                NativeMethodsShared.GetModuleFileName(NativeMethodsShared.NullHandleRef, sb, sb.Capacity);
-#else
-                NativeMethodsShared.GetModuleFileName(IntPtr.Zero, sb, sb.Capacity);
-#endif
-                path = sb.ToString();
-            }
+            var path = Path.Combine(
+                Path.GetDirectoryName(FileUtilities.ExecutingAssemblyPath)
+                    .TrimEnd(new[] { Path.DirectorySeparatorChar, Path.AltDirectorySeparatorChar }),
+                "MSBuild.exe");
```

Comment:
Probably worth looking at 864bde6c in its totality.

---
In reply to: [69628944](https://github.com/Microsoft/msbuild/pull/759#discussion_r69628944) [](ancestors = 69628944)